network-api/delete_non_staff.py

```diff
@@ -0,0 +1,33 @@
+"""Python script running with Heroku Scheduler add-on: check for accounts created by non-staff people and delete
+them."""
+
+import os
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "networkapi.settings")
+django.setup()
+
+
+def delete_non_staff():
+    from django.contrib.auth.models import User, Group
+    from django.db.models import Q
+
+    print("Deleting non staff users")
+    group_q = Group.objects.all()
+    non_staff = User.objects.exclude(
+        Q(email__contains='mozillafoundation.org') |
```

Comment:
this should probably be `Q(email__endswith='@mozillafoundation.org') |`, which will ensure sneaky people don't get by with something like "trollolol+mozillafoundation.org@example.com" - which seems unlikely, but lets just protect against it anyways.

Reply:
I am totally going to start using that now.

Reply:
You're totally right, let's fix that XD